docs/application/dotnet/guides/machine-learning/singleshot.md

```diff
@@ -43,33 +44,37 @@ To enable your application to use the Machine Learning Inference API functionali
     ```
 
 <a name="manage"></a>
-## Managing Tensor Information
+## Managing tensor information
```

Comment:
Global Comment: 
Thank you for updating the headings in sentence case.

It is recommended to use "sentence case" and "verb format" for headings.
Please do not use the gerund form (ing form).
For example, 
Please change:
## Managing Tensor Information
to
## Manage tensor information

For more information, see https://github.com/Samsung/tizen-docs/pull/1079#discussion_r408026272.